csbuild/toolchain/toolchain.py

```diff
@@ -905,7 +905,7 @@ def _runMultiFunc(*args, **kwargs):
 												break
 
 										assert func is not None, "this shouldn't happen"
-										if isinstance(func, (types.FunctionType, types.MethodType, property)):
+										if isinstance(func, (types.FunctionType, types.MethodType, property)): # pylint: disable=bad-option-value,no-else-raise
```

Comment:
`elif` -> `if`. Why is it complaining about bad-option-value here?

Reply:
The `bad-option-value` you see in a few places is the workaround I mentioned to make this pass with pylint 1.9.5 on python 2.  The `no-else-raise` doesn't exist that far back, so it flags the `bad-option-value` error on that, but doing it this way suppresses it.  Just FYI.

Reply:
Ah, ok. That makes sense. Well then when we fix all the elifs, we'll get rid of those too.